#! /bin/sh /usr/share/dpatch/dpatch-run
## 31_check_disk_local_option.dpatch by Jan Wagner <waja@cyconet.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix local option for check_disk

@DPATCH@
diff -Nur nagios-plugins-1.4.11/plugins/check_disk.c.orig nagios-plugins-1.4.11/plugins/check_disk.c
--- nagios-plugins-1.4.11/plugins/check_disk.c.orig	2007-12-08 17:34:05.000000000 +0100
+++ nagios-plugins-1.4.11/plugins/check_disk.c	2008-02-05 22:13:43.397159697 +0100
@@ -475,7 +475,7 @@
     {"iwarning", required_argument, 0, 'W'},
     /* Dang, -C is taken. We might want to reshuffle this. */
     {"icritical", required_argument, 0, 'K'},
-    {"local", required_argument, 0, 'l'},
+    {"local", no_argument, 0, 'l'},
     {"stat-remote-fs", required_argument, 0, 'L'},
     {"kilobytes", required_argument, 0, 'k'},
     {"megabytes", required_argument, 0, 'm'},
